src/atsu/common/src/messages/WeatherMessage.ts

```diff
@@ -61,4 +61,11 @@ export class WeatherMessage extends AtsuMessage {
 
         return message;
     }
+
+    public deserialize(jsonData: Record<string, unknown>): void {
```

Comment:
I would generally make those a static method instead, for example:

```ts
public static deserialize(jsonData: Record<string, unknown>): WeatherMessage
```

Reply:
fixed